backend_new/README.md

```diff
@@ -0,0 +1,29 @@
+# Setup
+## Installation
+Make sure the .env file's db placement is what works for your set up, Then run the following:
```

Comment:
Can you expand on this? I'm struggling to get the db setup command to work since I don't know what the format of the .env file should look like.